fp.js

```diff
@@ -18,17 +24,36 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
+/*
 people.forEach(function(person) {
   if(person.age < 35){
     peopleYoungerThan35.push(person)
   }
 });
+*/
+ people.filter(people=>{
```

Comment:
Like `.map`, with `.filter` you do not need to push into an empty array. Instead, you can save the result of filter directly in a variable. Filter's return statement should be whatever condition you want all the items in the resulting array to match. So the solution would look something like this...

```js
var peopleYoungerThan35 = people.filter(person => {
  return person.age < 35  // this means we want to only return items in `people` that have an age under 35
})
```